### ISSUE TITLE
Please could you add support for Bluetooth Govee lights?

### ISSUE THREAD

**Issue by umsie121** [bwp91/homebridge-govee#39]
* **Please explain your feature request in a one or two sentences.**

Hi please could you add support for Govee Bluetooth LED lights. I found this but I don’t know how to bring it in to Homebridge. 

https://www.npmjs.com/package/govee-led-client

Would really love for your plug-in to add this feature, or for some guidance for how to do it myself!

* **Is your feature request related to a problem? Please describe.**

* **Any particular ECOVACS devices that this relates to?**

* **Anything else?**

**Comment by bwp91** [comment]
Hi @umsie121 thanks for your post.

Unfortunately this is something that I don't plan to implement into this plugin due to a number of reasons, including:

* This would require extra packages to be installed outside of Homebridge
* The `govee-led-client` which you mentioned uses different packages depending on platform (ie Windows or Not Windows) which would be complicated to implement in a plugin that should run on all systems
* There is no way of requesting the current state of a device, meaning that if any lights were controlled through the Govee app, then HomeKit would continue to display it's previous state
* This would require a user to obtain the mac address of their Govee lights which is possible but not straightforward

In short there is just _so_ much that could go wrong for a user and doesn't equate with my idea of how a Homebridge plugin should be!

**Comment by bwp91** [comment]
Hi @umsie121 

Initial support for bluetooth is now available in the beta version of the plugin… happy to walk you through the steps to test if you’re interested!

**Comment by umsie121** [comment]
Hey Ben!

Thanks so much for remembering. I’d love more info

PS: it’s coming home(kit)


On 11 Jul 2021, at 7:29 pm, Ben ***@***.***> wrote:

﻿

Hi @umsie121<https://github.com/umsie121>

Initial support for bluetooth is now available in the beta version of the plugin… happy to walk you through the steps to test if you’re interested!

**Comment by bwp91** [comment]
Could I message you on Discord?

https://discord.gg/cMGhNtZ3tW
